target/linux/ipq40xx/patches-5.4/706-dt-bindings-net-add-QCA807x-PHY.patch

```diff
@@ -0,0 +1,61 @@
+From c66863c1ba8995b61e6d727d78a241c734f5bb57 Mon Sep 17 00:00:00 2001
```

Comment:
Is there a reason why you did this as patch instead a file like you did for qca807x.c?

Reply:
It's simply as it requires no changes that are OpenWrt specific, unlike the PHY driver.